non_distinct_isins.csv

```diff
@@ -0,0 +1,237 @@
+company_id,bloomberg_id,company_name,country_of_domicile,ticker,corporate_bond_ticker,figi,isin,cusip,sedol1,asset_type,security_icb_subsector,security_bics_subgroup,security_bclass4,security_mapped_sector,cfi_code,security_type,maturity_date,issue_date,amount_issued,amount_out,coupon_value,par_price,par_amount,current_face_amount,accrued_interest,security_amount_out_native,unit_share_price,position_market,market_value,current_shares_outstanding,current_shares_outstanding_all_classes,free_float_percent,free_float_shares,exchange_rate_usd,moody_rating,s_and_p_rating,fitch_rating,financial_timestamp
```

Comment:
This repo is public, and this financial data seems to me that it should be private/ not committed to version control lol... 
@Clare2D thoughts?

Reply:
I totally agree, I just don't see how financial data can be entirely kept out of this as long as it is not solved on DataStore.. Of course, I can reduce the list to only ISINs, but I'm not sure how much better that is. Open to suggestions here

Reply:
ISIN only makes sense here as a better alternative!

Reply:
Hmm. If you could put the dataset on dropbox somewhere (maybe in the project directory?). You could:
* .gitignore this file, so as to not commit it to version control
* access the file using `rdrop2` and an oauth token (see #50)

There would just need to be a sensible place on dropbox to upload this file, @Clare2D thoughts?

Reply:
This is a temporary solution only and needs to be used by everyone using the code, so we can't ignore it. Maybe as a longer solution we can start to think about the rdrop2, but I don't think now is the best time to start on that.. but eventually that could work for a few different things.

Reply:
Ah understood. I thought this was "project specific", didn't realize everyone would need the fix. 
I'll leave it to you to decide then!

Reply:
Ahh the actual solution to this should be that this fix happens in data_preparation not in PACTA_analysis. Let's get to the solution with Taylor asap and see when we can write the test into data prep and remove this solution!